h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -57,12 +57,6 @@
      */
     double time_remaining_secs();
 
-    /**
-     * Inform the Iterator that a model build failed in case it needs to adjust its internal state.
-     * @param failedModel
-     */
-    void modelFailed(Model failedModel);
```

Comment:
it is package private. As we moved  early stopping logic outside of the iterator we don't do anything in `modelFailed` method.